vendor/github.com/soheilhy/cmux/matchers.go

```diff
@@ -44,13 +44,13 @@ func prefixByteMatcher(list ...[]byte) Matcher {
 }
 
 var defaultHTTPMethods = []string{
-	"OPTIONS",
+	// "OPTIONS",
```

Comment:
This change might just work, but we need to limit our changes to our own code base--nothing from the vendor directory.

Reply:
It is the only place where OPTIONS or TRACE are declared so I figure this might be the place

Reply:
Agree, we should not be updating vendor code.